app/assets/javascripts/Patient.js

```diff
@@ -0,0 +1,143 @@
+var Schema = mongoose.Schema;
+var Code = Schema.Types.Code;
+var Interval = Schema.Types.Interval;
+var Quantity = Schema.Types.Quantity;
+
+var PatientSchema = new Schema({
+  birth_datetime: DateTime,
+  qdm_version: { type: String, default: "5.3" },
+  ethnicity: Code,
+  race: Code,
+  sex: Code,
```

Comment:
Does "payer" and "expired" need to be listed here?

Reply:
Like `ethnicity`, `race`, and `sex` - `payer` and `expired` are also actual QDM datatypes, which means there are models for them.

I think we decided that `ethnicity`, `race`, and `sex` should be on the patient itself as well, because that is similar to how HDS did things.

So I guess my question is, do we want to do this for _ALL_ of the QDM patient characteristic types? I'm not opposed to that, I just want to clarify that there is duplication by doing this.

An alternative might be using the patient characteristic datatypes themselves (so initialize them using the QDM models, and put them inside the `history_elements`), rather than sticking that information on the patient model itself, and instead have helper methods (like `patient.procedures`, `patient.encounters`, etc) that you can call on the patient (i.e. `patient.race`), which looks into `history_elements`, grabs any existing data criteria for race, and returns what it finds.

Reply:
i like the alternative

Reply:
if we do the alternative, does it make sense to rename "history elements" to something else? don't have many ideas here - maybe "attributes"?

Reply:
Right, we had a long discussion on slack about how to name it, and I don't think any of us were too attached to any of the options. I do agree it should be changed with the understanding that we are going with the "alternative" above.

Attributes sounds nice, but unfortunately if we used that here it would be overloading the term, as the QDM spec uses it to describe fields on QDM datatypes.

We also considered: `entries`, `data_criteria`, `data_criteria_entries`, `patient_history_elements`, `patient_history_entries`